SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2674,6 +2665,36 @@ bool USpatialNetDriver::IsReady() const
 	return bIsReadyToStart;
 }
 
+bool USpatialNetDriver::IsLogged(Worker_EntityId ActorEntityId, EActorMigrationResult ActorMigrationFailure)
+{
+	// Clear the log migration store at the specified interval
+	const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
+	if (HasTimedOut(Settings->ActorMigrationLogRate, MigrationTimestamp))
+	{
+		MigrationFailureLogStore.Empty();
+	}
+
+	// Check if the pair of actor and failure reason have already been logged
+	return MigrationFailureLogStore.FindPair(ActorEntityId, ActorMigrationFailure) != nullptr;
+}
+
+void USpatialNetDriver::AddLogRecord(Worker_EntityId ActorEntityId, EActorMigrationResult ActorMigrationFailure)
```

Comment:
The AddLogRecord functionality can be incorporated into the IsLogged function. I will make this change Monday morning as my PC is currently rebuilding the engine.

Reply:
Done